nx/lib/nx/serving.ex

```diff
@@ -674,13 +715,13 @@ defmodule Nx.Serving do
     {__MODULE__, name}
   end
 
-  defp handle_init(module, type, arg) do
-    case module.init(type, arg) do
+  defp handle_init(module, type, arg, opts) do
+    case module.init(type, arg, opts) do
```

Comment:
Should we check if /2 is exported and do soft deprecation instead?

Reply:
Nah, i think it is fine with breaking for now. Most people are likely using the function based version which I will keep as process options.

Reply:
Makes sense!